site/src/components/Home.js

```diff
@@ -65,7 +65,7 @@ class HomePage extends Component {
 		// To fetch reactive search github stars
 		fetch('https://api.github.com/repos/appbaseio/reactivesearch')
 			.then(res => res.json())
-			.then(res => {
+			.then((res) => {
```

Comment:
Why there are changes in this file?

Reply:
I back merged `next` and tried to commit, and it was throwing `eslint` errors for the above files, so had to fix it